packages/smart-wallet/test/test-addAsset.js

```diff
@@ -87,3 +99,441 @@ test('avoid O(wallets) storage writes for a new asset', async t => {
   t.is(base.addedWrites, 0);
   t.is(exp.addedWrites, 0);
 });
+
+const BOARD_AUX = 'boardAux';
+/**
+ * Make a storage node for auxilliary data for a value on the board.
+ *
+ * @param {ERef<StorageNode>} chainStorage
+ * @param {string} boardId
+ */
+const makeBoardAuxNode = async (chainStorage, boardId) => {
+  const boardAux = E(chainStorage).makeChildNode(BOARD_AUX);
+  return E(boardAux).makeChildNode(boardId);
+};
+
+const marshalData = makeMarshal(_val => Fail`data only`);
+
+/** @type {<T>(x: Promise<T | null>) => Promise<T>} */
+const the = async x => {
+  const res = await x;
+  assert(res, 'resolution is unexpectedly falsy');
+  return res;
+};
+
+/** @typedef {import('@agoric/internal/src/storage-test-utils.js').MockChainStorageRoot} MockVStorageRoot */
+
+const UNIT6 = 1_000_000n; // IST has six decimals
```

Comment:
consider `IST_UNIT` to make the name more self-documenting
```suggestion
const IST_UNIT = 1_000_000n;
```
([precedent](https://github.com/Agoric/agoric-sdk/blob/0d7b599fa3dafc827c560a140c542168e755b59a/packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/param-change-offer-gen.mjs#L5))